src/bin/pg_autoctl/keeper_pg_init.c

```diff
@@ -693,7 +695,7 @@ create_database_and_extension(Keeper *keeper)
 								   pgSetup->dbname,
 								   config->nodename,
 								   pg_setup_get_username(pgSetup),
-								   "trust",
+								   pg_setup_get_auth_method(pgSetup),
```

Comment:
function returns "trust" as default value

Reply:
Indeed. And when the user said `--auth cert` or `--auth scram-sha-256`, that's what we're going to use here when opening the LAN for a whole formation of nodes.